packages/governance/src/contractGovernorKit.js

```diff
@@ -1,16 +1,57 @@
 import { Fail } from '@agoric/assert';
-import { makeTracer } from '@agoric/internal';
-import { prepareExoClassKit } from '@agoric/vat-data';
+import { UnguardedHelperI, makeTracer } from '@agoric/internal';
+import { M, prepareExoClassKit } from '@agoric/vat-data';
 import { E } from '@endo/eventual-send';
+import {
+  InvitationShape,
+  InstanceHandleShape,
+  InstallationShape,
+} from '@agoric/zoe/src/typeGuards.js';
+import { TimestampShape } from '@agoric/time';
 import { setupApiGovernance } from './contractGovernance/governApi.js';
 import { setupFilterGovernance } from './contractGovernance/governFilter.js';
 import {
   CONTRACT_ELECTORATE,
   setupParamGovernance,
 } from './contractGovernance/governParam.js';
+import { ClosingRuleShape, ParamChangesSpecShape } from './typeGuards.js';
 
 const trace = makeTracer('CGK', false);
 
+const ContractGovernorKitI = {
+  helper: UnguardedHelperI,
+  creator: M.interface('Contract Governor Kit creator', {
+    replaceElectorate: M.call(InvitationShape).returns(M.promise()),
+    voteOnParamChanges: M.call(
+      InstallationShape,
+      TimestampShape,
+      ParamChangesSpecShape,
+    ).returns(M.promise()),
+    voteOnApiInvocation: M.call(
+      M.string(),
+      M.arrayOf(M.any()),
+      InstallationShape,
+      TimestampShape,
+    ).returns(M.promise()),
+    voteOnOfferFilter: M.call(
+      InvitationShape,
+      TimestampShape,
+      M.arrayOf(M.string()),
+    ).returns(M.promise()),
```

Comment:
voteOnOfferFilter(voteCounterInstallation, deadline, strings) {
```suggestion
    voteOnOfferFilter: M.call(
      InstallationShape,
      TimestampShape,
      M.arrayOf(M.string()),
    ).returns(M.promise()),
```

Reply:
Thanks for catching that. CI didn't. Would that be possible? We have test coverage but apparently these two are interchangeable?
```js
export const InvitationShape = M.remotable('Invitation');
export const InstallationShape = M.remotable('Installation');
```

Reply:
Ah, that label is purely diagnostic.
```
 * @property {(label?: string) => Matcher} remotable
 * Matches a far object or its remote presence.
 * The optional `label` is purely for diagnostic purposes and does not
 * add any constraints.
```